Library/Formula/phantomjs.rb

```diff
@@ -17,9 +17,13 @@ class Phantomjs < Formula
   depends_on "openssl"
 
   def install
-    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
-              "QT_CFG+=' -openssl -openssl-linked -I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}'"
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs
+    if build.stable?
+      system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
+             "--qt-config", "-I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}"
+    else
+      inreplace "build.py", "/usr/local", HOMEBREW_PREFIX
```

Comment:
@Vitallium we can't rely on the hard-coded `/usr/local` since Homebrew can be installed elsewhere, but `--qt-config` wasn't working on `master`:
```Running configuration tests...

Could not determine the target architecture!
Turn on verbose messaging (-v) to see the final report.
OpenSSL support cannot be enabled due to functionality tests!
 Turn on verbose messaging (-v) to /private/tmp/phantomjs20160110-52882-1iwjqoe/src/qt/qtbase/configure to see the final report.
 If you believe this message is in error you may use the continue
 switch (-continue) to /private/tmp/phantomjs20160110-52882-1iwjqoe/src/qt/qtbase/configure to continue.

ERROR: Failed to build PhantomJS! Configuration of Qt Base failed.
```
Any ideas?